docs/how-does-it-work/xp.md

```diff
@@ -8,7 +8,7 @@ title: 🧬 XP & Seeds 🌱
 
 ## **📊 How does XP work?**
 
-Whether it’s completing random quests, sharing things on [Discord](https://discord.gg/ZqdPP9b), writing useful stuff on [Discourse](https://metagame.trydiscourse.com/), or laying down code on [GitHub](https://github.com/metafam/) - your XP is automagically being accrued.
+Whether it’s completing random quests, sharing things on [Discord](https://discord.gg/ZqdPP9b), writing useful stuff on [Discourse](https://forum.metagame.wtf/), or laying down code on [GitHub](https://github.com/metafam/) - your XP is automagically being accrued.
```

Comment:
Thanks for this. It actually made me realise that after 4 months of MG I'd not set my Discourse username. :laughing:

Reply:
@luxumbra  😆